specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -0,0 +1,400 @@
+# Push Notifications in Project Reunion SDK
+
+# Background
+
+The Windows Push Notification Services (WNS) enable third-party developers to send toast, tile, badge, and raw updates from their own cloud service. This provides a mechanism to deliver new updates to your users in a power-efficient and dependable way. For example, the skype app service can triggering an incoming call toast by Pushing a Payload to the WNS service. Push is generally used for time critical events that require immediate user/developer action. 
+
+* [The Push notification WinRT Public APIs](https://docs.microsoft.com/en-us/uwp/api/windows.networking.pushnotifications?view=winrt-19041)
+  Defines the classes that encapsulate Push notifications. (UWP and Win32)
+* [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview)
+
+## Push Flow:
+In order to subscribe to Push Notifications, an app first needs to request a Push Channel with WPN client which is essentially an Http Endpoint that it can post notifications to. 
```

Comment:
What's WPN? Above seemed to call this WPS.

Reply:
DISCUSSION: WPN is the "Windows Push Notification" API surface.  WPS is an internal term for the "Windows Push Services" - we normally call this "WNS" of "Windows Notification Services."

RECOMMEND: Split the API spec doc into the parts for documentation (and API) and the details of how we do stuff.